### DISCUSSION TITLE
What is best practice for alerting screen readers of current slide content when using prev/next swiper navigation?

### DISCUSSION THREAD

**Discussion by nateplusplus** [nolimits4web/swiper#4063] [Q&A]
Hello! My team wants to make sure that our swiper experience is not degraded for visually impaired users. What we have come to understand is that the content within the current slide should be announced to the screen reader when navigating without changing focus from navigation buttons, as mentioned by [W3C guidelines here](https://www.w3.org/WAI/tutorials/carousels/functionality/#announce-the-current-item).

Does SwiperJS already do this somehow under the hood, or is there something we can set up to have it do this? Has anyone explored this?

Thanks in advance for any tips or resources for further learning!

**Comment by megatux** [comment]
I want to understand this behavior, too. I'm using Orca screen reader and do not hear any events on slide changes, a11y module is ON.
